Icolbutler 419 1

hapaxlegomenon opened the pull request:
@Spunkle Please double check 419 in Oxygen, but otherwise good to go! Hurrah!

hapaxlegomenon committed:
Create icolbutler-419

Uploaded 419 from Dropbox folder. Will work on this/finalize edit (it looks pretty good already).

hapaxlegomenon committed:
Update icolbutler-419 with copyright date

from 2012 to 2015 under creative commons

hapaxlegomenon committed:
Update/finalized icolbutler-419

Finalized all edits (fixed boilerplate for history, deleted cm)

Spunkle commented:
Great stuff here, @hapaxlegomenon!

Yep, this validates on oxygen with our schema and gets a solid *imprimatur* from me so please feel free to merge, delete branch, etc.

The only feedback I have is very minor / philosophical (almost not worth mentioning), and it's completely up to you what if anything to do with it:
* is there a difference between *libertabusq(ue)* and *posteris / q(ue)* -- I mean in terms of where the expan and abbrev elements begin (i.e. before the noun or before the enclitic)?
* another question of consistency: the longer free-text handNote is wrapped in a paragraph element, but not the shorter handNotes about each line height; also only the range is coded min="X" max="Y" but not absolute values e.g. extent="Z". I honestly don't care about this (otherwise I would check on TEI.org, EpiDoc guidelines, etc.) but just flag this in case you are interested in that.
* more about content now: it never occurred to me that the freedmen & -women might belong to the *children*, but I'm embarrassed that I can't say for certain that they're definitely the mother's, because I don't know much about Roman inheritance laws, *officia* owing upon *liberti* etc., and I've never tracked this when reading translations or commentaries of inscriptions. It seems to me like something we could commit to in the translation though (i.e. either "their" or "her" but not either / or). My gut instinct is that this is a formula of course, but originally depended upon the wishes of the deceased, rather than the living people making the dedication, since they are probably producing the monument in accordance with the will of the deceased, and they themselves are of course free to change their mind about their inheritance at any moment before they die. But I agree with you that the Oxford comma + "their descendants" hedges perfectly the ambiguity behind *posteris eorum*.

But again -- this looks really good, and thanks for putting in the hard work on the template.

hapaxlegomenon committed:
Update icolbutler 419-1

Fixed <expan> problem, finalized translation (with her/their issue), and removed unnecessary <p> elements

hapaxlegomenon commented:
Fixed a couple of the problems @Spunkle mentioned; should be good to go!!!!
